src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -345,7 +363,7 @@ private void onVersionChanged() {
   private void updateVersionTextIfCurrent(@NotNull FlutterSdk sdk, @NotNull String value) {
     final FlutterSdk current = FlutterSdk.forPath(getSdkPathText());
     if (current == null) {
-      myVersionLabel.setText("");
+      myVersionLabel.setText(" ");
```

Comment:
maybe comment why a " " is used instead of the empty string?

Reply:
I'll add a comment (it's so the layout doesn't collapse to 0 height).